test/config/integration/mysql_test_config.yaml

```diff
@@ -0,0 +1,30 @@
+admin:
```

Comment:
Please move this file into the extensions directory alongside the extension tests.